simtools/util/general.py

```diff
@@ -770,3 +775,72 @@ def get_log_excerpt(log_file, n_last_lines=30):
         f"{collect_final_lines(log_file, n_last_lines)}\n\n"
         "====================================================================\n"
     )
+
+
+def convert_2D_to_radial_distr(xaxis, yaxis, hist2d, bin_size=50, max_dist=1000):
+    """
+    Convert a 2D histogram of the photon positions on the ground to a 1D radial distribution.
```

Comment:
I changed the docstring a bit but please keep in mind that the function has to deal with **position** since this is the only intended for it (and we use `np.sqrt(x_element**2 + y_element**2)`). it could position of particles, telescopes, etc...